mist/master/src/main/scala/io/hydrosphere/mist/master/interfaces/http/WSApi.scala

```diff
@@ -19,23 +19,25 @@ class WSApi(streamer: EventsStreamer) {
   import JsonCodecs._
 
   val route = {
-    path( "v2" / "api" / "ws" / "all" ) {
-      get {
-        handleWebSocketMessages(allEventsWsFlow())
+    pathPrefix("v2" / "api"/ "ws" ) { parameter('withLogs ? true)  { withLogs =>
```

Comment:
I think it will be better to turn logs off by default. I suppose there aren't a lot of users who use WebSockets.

Reply:
The main purpose of this is not breaking back compatibility.

Reply:
We can easily fix it for ui. WebSocket interface wasn't previously declared as public or stable interface

Reply:
Ok, I'll fix then